src/defaultSelector.py

```diff
@@ -57,8 +70,149 @@ def select(self, los, geometry):
         # obavijesti sve zainteresirane da je selekcija promijenjena
         Signals.get().selectionChanged.emit(si)
 
-    def getMeshInterscection(self, K, P0, mesh: om.TriMesh):
+    def getMeshInterscectionSDBTNew(self, ray: dmnsn_ray, fhlist, points, fv_indices):
```

Comment:
sorry for nitpickink, but the name of the function is bit cryptic. What does SDBTNew mean?

Reply:
SDBT is a shortcut for SubDixBoxTree, I fixed the function name by removing the shortcut